tests/integration/test_groupnameupdate.py

```diff
@@ -24,9 +25,9 @@ def setUp(self):
         ("foreign_as_regular_user"     , "user_a"          , "group_y_id"   , "fancy_group"  , 403),
         ("admin_use_existing_groupname", "admin"           , "group_x_id"   , "group_y"      , 400),
         ("unauthorized"                , ""                , "group_x_id"   , "fancy_group"  , 401),
-        ("admin_expired_token"         , "admin:expired"   , "group_x_id"   , "fancy_group"  , 401),
+        ("admin_expired_token"         , "admin: expired"   , "group_x_id"   , "fancy_group"  , 401),
```

Comment:
```suggestion
        ("admin_expired_token"         , "admin:expired"   , "group_x_id"   , "fancy_group"  , 401),
```
This needs to stay, otherwise it screws up the test.

Reply:
why this thing checks within strings is a conundrum... :/

Reply:
It doesn't, I search+replace fixed the errors manually. Good catch. Test still passed though 🤔